pkg/discovery/worker/garbage_collector.go

```diff
@@ -110,3 +156,78 @@ func (g *GarbageCollector) isLeakedPod(pod api.Pod) bool {
 
 	return false
 }
+
+// The controllers cleanup is supposed to happen only at the startup
+// We fail in case of errors to force kubeturbo to restart and try the cleanup again
+func (g *GarbageCollector) CleanupControllers() error {
```

Comment:
Suggest renaming to something like `revertControllers` or `resetControllers`, or some other name to indicate that we are reverting back to the original state.

Reply:
Done!